src/Datadog.Trace.ClrProfiler.Managed.Loader/FileSink.cs

```diff
@@ -0,0 +1,56 @@
+//------------------------------------------------------------------------------
+// <auto-generated />
+// This file was automatically generated by the UpdateVendors tool.
+//------------------------------------------------------------------------------
```

Comment:
Is this file actually actually auto-generated? If not, we should remove this header.

Reply:
It tricks style cop into allowing the conventions in Serilog.
I can add a comment that is more clear?

Reply:
Yeah, thanks. Maybe replace `This file was automatically generated by the UpdateVendors tool` with something like `Tagged as auto-generated to skip StyleCop validation`.